### ISSUE TITLE
Emphasis inline with HTML Entities

### ISSUE THREAD

**Issue by RickStrahl** [lunet-io/markdig#97]
The emphasis extension has an issue when trying to emphasise an Html entity:

```markdown
### Works:
MyBrand^®^ and MyTrademark^™^  

This is text MyBrand ^&reg;^ and MyTrademark ^&trade;^

This is text MyBrand^®^ and MyCopyright~©~
This is text MyBrand^12^

### Doesn't work
This is text MyBrand^&reg;^ and MyTrademark^&trade;^

This is text MyBrand~&reg;~ and MyCopyright^&copy;^
```

which produces:

![image](https://cloud.githubusercontent.com/assets/1374013/23281502/a2aa4ac4-f9c1-11e6-956f-643e7bdbcf40.png)

I was going to take a look at fixing this but the project doesn't work out of box. Yeah I know DNX and project.json versioning stuff 😢

**Comment by xoofx** [comment]
Indeed, according to [babelmark](https://babelmark.github.io/?text=%23%23%23+Works%3A%0AMyBrand%5E%C2%AE%5E+and+MyTrademark%5E%E2%84%A2%5E++%0A%0AThis+is+text+MyBrand+%5E%26reg%3B%5E+and+MyTrademark+%5E%26trade%3B%5E%0A%0AThis+is+text+MyBrand%5E%C2%AE%5E+and+MyCopyright~%C2%A9~%0AThis+is+text+MyBrand%5E12%5E%0A%0A%23%23%23+Doesn%27t+work%0AThis+is+text+MyBrand%5E%26reg%3B%5E+and+MyTrademark%5E%26trade%3B%5E%0A%0AThis+is+text+MyBrand~%26reg%3B~+and+MyCopyright%5E%26copy%3B%5E) 
It seems that only Pandoc is supporting this correctly... will try to check why...

**Comment by RickStrahl** [comment]
I didn't know about babelmark. That's sweet!

**Comment by xoofx** [comment]
This is fixed by b47ae8d 
The nuget 0.10.6 is on its way

**Comment by RickStrahl** [comment]
Checked out in 0.10.6 and all looks good. Thanks for the quick turnaround and getting the NuGet package out - that's awesome!
